File: problems/0/0/2/002.lua
-- Copyright (C) 2014 Jorge Aparicio

ans = 0
curr = 1
next = 2

while curr < 4000000 do
  if curr % 2 == 0 then
    ans = ans + curr
  end

  tmp = next
  next = next + curr
  curr = tmp
end

print(ans)

File: problems/0/0/2/002.rb
# Copyright (C) Jorge Aparicio

ans = 0
curr = 1
nxt = 2

while curr < 4000000 do
  if curr % 2 == 0 then
    ans += curr
  end

  tmp = nxt
  nxt += curr
  curr = tmp
end

puts ans

